client/ayon_houdini/plugins/create/create_review.py

```diff
@@ -150,3 +151,10 @@ def get_pre_create_attr_defs(self):
                       minimum=0.0001,
                       decimals=3)
         ]
+
+
+class CreateFlipbookReview(CreateReview):
+    
+    identifier = "io.ayon.creators.houdini.review.flipbook"
+    label = "Review (Flipbook)"
+    node_type = "flipbook"
```

Comment:
> Client mentioned that for them it made sense to set a default in settings somewhere so that e.g. a review defaults to either the OpenGL node OR the Flipbook node. I'm not convinced that's the best approach yet - I think having a separate creator may be safest, that is _also_ generating `review` product type should be just fine. Then in settings the studio can enable/disable EITHER creator so the old remains as is, and the new becomes available. Since we may never know how much the two nodes will remain aligning.

Quoted from the linked issue. 
Now, should we have a separate creator for the flipbook node? or using the same creator while exposing the node type?

Reply:
Whatever makes most sense from development side of things I'd say. It may appear cleaner if it's one Creator to avoid clutter in the Create list?

Reply:
https://github.com/ynput/ayon-houdini/pull/167/commits/5f8d3254c8b611c627e81f2865608637aa605fbb